conf/config.level1.neon

```diff
@@ -9,8 +9,17 @@ parameters:
 
 rules:
 	- PHPStan\Rules\Classes\UnusedConstructorParametersRule
-	- PHPStan\Rules\Constants\ConstantRule
 	- PHPStan\Rules\Functions\UnusedClosureUsesRule
 	- PHPStan\Rules\Variables\EmptyRule
 	- PHPStan\Rules\Variables\IssetRule
 	- PHPStan\Rules\Variables\NullCoalesceRule
+
+services:
+	-
+		class: PHPStan\Rules\Constants\ConstantRule
+		factory: @PHPStan\Rules\Constants\ConstantRuleFactory::create()
```

Comment:
I took inspiration from `ConstantResolverFactory` to create this factory which uses the same way to retrieve `%dynamicConstantNames%`.

My first try was passing `%dynamicConstantNames%` directly via `arguments:` but this didn't work.